Content/tasks.tests.unit.cake

```diff
@@ -57,9 +57,11 @@ Sitecore.Tasks.RunServerUnitTestsTask = Task("Unit Tests :: Run Server Tests")
 
         var _openCoverResultsFilePath = new FilePath($"{Sitecore.Parameters.XUnitTestsCoverageOutputDir}/coverage.xml");
 
+        var parallelismOptionArgument = Sitecore.Parameters.XUnitTestsRunInParallel ? ParallelismOption.All : ParallelismOption.None;
+        Information($"Parallelism Option Argument = {parallelismOptionArgument}");
```

Comment:
Please remove this info log

Reply:
https://cakebuild.net/api/Cake.Common.Tools.XUnit/ParallelismOption/ this enum contains 4 values. I think it makes sense to have all options available, to not change it in the future again

Reply:
> https://cakebuild.net/api/Cake.Common.Tools.XUnit/ParallelismOption/ this enum contains 4 values. I think it makes sense to have all options available, to not change it in the future again

Good mention. Now we can pass any of the 4 arguments as a string. Is it fine?

Reply:
Approved